app/src/aiMembers/aiMembers.html

```diff
@@ -1,6 +1,106 @@
-<p>This is the partial for view 2.</p>
-<p>
-  Showing of 'interpolate' filter:
-  {{ 'Current version is v%VERSION%.' | interpolate }}
-</p>
-<div></div>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+<div class = "container-fluid">
+<div class="totalmembers" ng-controller="AiMembersCtrl">
+
+    <!-- <div class = "row"> -->
+    <input class = "members" id="tab1" type="radio" name="tabs" checked>
+    <label class = "members" for="tab1">Web Dev</label>
+
+    <input class = "members" id="tab2" type="radio" name="tabs">
+    <label class = "members" for="tab2">Smash Block</label>
+
+     <input class = "members" id="tab3" type="radio" name="tabs">
+    <label class = "members" for="tab3">Serenade</label>
+    <!-- </div> -->
+
+
+  <div class = "membersbound">
+      <div class = "row">
+          <div class="col-sm-5">
+    <div id="Web-Dev">
+      <div class="container">
+          <table class="table table-bordered table-hover" id="1" ng-repeat = "x in firsttable">
```

Comment:
id's should follow all coding naming conventions, so you shouldn't start one with a number or a capital.